tests/std/lit.site.cfg.in

```diff
@@ -25,9 +25,7 @@ lit_config.expected_results[config.name] = stl.test.file_parsing.parse_result_fi
 lit_config.include_dirs[config.name]     = \
     ['@STL_TESTED_HEADERS_DIR@', '@LIBCXX_SOURCE_DIR@/test/support', '@STL_SOURCE_DIR@/tests/std/include']
 lit_config.library_dirs[config.name]     = ['@CMAKE_LIBRARY_OUTPUT_DIRECTORY@', '@TOOLSET_LIB@']
-lit_config.test_subdirs[config.name]     = \
-    [os.path.normpath(os.path.join('@STD_TEST_SUBDIRS_ROOT@', path)) for \
-     path in stl.test.file_parsing.parse_commented_file('@STD_TEST_SUBDIRS_FILE@')]
+lit_config.test_subdirs[config.name]     = ['@CMAKE_CURRENT_SOURCE_DIR@/tests']

```

Comment:
As @CaseyCarter requested, this behavioral change should be recorded in the PR description.